src/main/java/com/spotify/docker/client/messages/NetworkCreation.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2014 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.docker.client.messages;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.MoreObjects;
+import com.google.common.collect.ImmutableList;
+
+import java.util.List;
+
+import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.ANY;
+import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.NONE;
+
+@JsonAutoDetect(fieldVisibility = ANY, getterVisibility = NONE, setterVisibility = NONE)
+public class NetworkCreation {
+
+  @JsonProperty("Id") private String id;
+  @JsonProperty("Warnings") private ImmutableList<String> warnings;
```

Comment:
Transformed in String

2015-12-28 18:29 GMT+01:00 David Xia <notifications@github.com>:

> In src/main/java/com/spotify/docker/client/messages/NetworkCreation.java
> <https://github.com/spotify/docker-client/pull/307#discussion_r48493055>:
>
> > +
> > +import com.fasterxml.jackson.annotation.JsonAutoDetect;
> > +import com.fasterxml.jackson.annotation.JsonProperty;
> > +import com.google.common.base.MoreObjects;
> > +import com.google.common.collect.ImmutableList;
> > +
> > +import java.util.List;
> > +
> > +import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.ANY;
> > +import static com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility.NONE;
> > +
> > +@JsonAutoDetect(fieldVisibility = ANY, getterVisibility = NONE, setterVisibility = NONE)
> > +public class NetworkCreation {
> > +
> > +  @JsonProperty("Id") private String id;
> > +  @JsonProperty("Warnings") private ImmutableList<String> warnings;
>
> Does this work? Docs here
> <https://docs.docker.com/engine/reference/api/docker_remote_api_v1.21/#create-a-network>
> imply it's Warning and just a String.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/spotify/docker-client/pull/307/files#r48493055>.
>



-- 
Damiano Pezzotti
...................................... . .
Software Architect | Alambitco
mobile +39 333 2116258
fax +39 02 700432209
email damy@alambitco.com
web www.alambitco.com
flickr www.flickr.com/photos/damianopezzotti
linkedin www.linkedin.com/in/damianopezzotti
pgp id 0x12DFD989